spec/views/users/show_spec.rb

```diff
@@ -15,13 +15,19 @@
                          text: 'Do more practice if your want to success in your life.')
     visit user_path(@author)
   end
-  it 'Shows Athor name' do
+  it 'Shows Author name' do
     expect(page).to have_content(@author.name)
   end
   it 'Shows Author biography' do
     expect(page.body).to have_content(@author.bio)
   end
 
```

Comment:
- [ ] _Tests that asserts you can see the user's first 3 posts_ is still missing. Let's include this missing test case.